src/room/LobbyView.tsx

```diff
@@ -116,12 +121,18 @@ export const LobbyView: FC<Props> = ({
         <div className={styles.content}>
           <VideoPreview matrixInfo={matrixInfo} muteStates={muteStates}>
             <Button
-              className={styles.join}
-              size="lg"
-              onClick={onEnter}
+              className={classNames(
+                "cpd-theme-dark",
```

Comment:
In the designs, this button doesn't appear to have dark mode forced on it

Reply:
It is white in both designs though: https://www.figma.com/file/AjvSw4lI9iGO5aTe4WAl4r/Element-Call?type=design&node-id=3901-72407&mode=design&t=vTwhO0KU481UpZud-0